Automattic/pocket-casts-ios
# Moved `Help & Support` to Profile screen

part of  #1234 

Please include a summary of the changes and the related issue.

## To test
1. go to Profile page and see `Help & Support` 
2. Tap on `Help & Support` and see that you get the support web view 
3. Got to Settings and see `Help & Support`  is removed.

https://github.com/Automattic/pocket-casts-ios/assets/1335657/fcf18314-6f1d-4f56-8899-2592d6b60681



Steps to test your PR.

## Checklist
